ci/docker/Dockerfile.build.ubuntu_cpu

```diff
@@ -48,9 +48,13 @@ COPY install/ubuntu_docs.sh /work/
 RUN /work/ubuntu_docs.sh
 
 ARG USER_ID=0
+ARG GROUP_ID=0
 COPY install/ubuntu_adduser.sh /work/
 RUN /work/ubuntu_adduser.sh
 
+COPY install/ubuntu_runas_sudo.sh /work/
```

Comment:
Please don't add this. We specifically don't want to run with sudo

Reply:
good catch